test/shared-class.test.js

```diff
@@ -198,6 +198,25 @@ describe('SharedClass', function() {
     });
   });
 
+  describe('sharedClass.findMethodByName()', function() {
+    var sc;
+    var sm;
+    it('finds sharedMethod by prototype name', function() {
+      sc = new SharedClass('SomeClass', SomeClass);
+      var PROTOTYPE_METHOD_NAME = 'SomeClass.prototype.testMethod';
+      sm = sc.defineMethod(PROTOTYPE_METHOD_NAME);
+      assert(sc.findMethodByName('SomeClass.prototype.testMethod') === sm);
```

Comment:
Either use PROTOTYPE_METHOD_NAME here too, or else get rid (inline all usages) of PROTOTYPE_METHOD_NAME.

I would personally not bother with defining a constant for this string.